src/did/did.ts

```diff
@@ -82,7 +86,10 @@ class DIDDocument implements Did {
         break;
       }
       case IKeyType.EcdsaSecp256k1RecoveryMethod2020: {
-        this.context = [constant['DID_' + keyType].DID_BASE_CONTEXT];
+        this.context = [
+          constant['DID_' + keyType].DID_BASE_CONTEXT,
+          constant['DID_' + keyType].DID_KEYAGREEMENT_CONTEXT,
```

Comment:
You should not take DID_KEYAGREEMENT_CONTEXT context here since you ar enot allowing keyagreement verrelationship